lblod/ember-rdfa-editor-lblod-plugins
# GN-4479: make address node work properly inside a location variable

### Overview
This PR adds support for inserting/editing address variables inside location variables. 
Specifically, it includes the following changes:
- The address node-view has been added to the embedded-editor of location variables
- Several css changes to ensure it remains clear to the user which variable is selected


##### connected issues and PRs:
[GN-4479](https://binnenland.atlassian.net/browse/GN-4479?atlOrigin=eyJpIjoiYWVmMTZlNzhhODdjNDZhZTk4YmNkYzJjY2ZiZDhlZWUiLCJwIjoiaiJ9)

### How to test/reproduce
- Open the regulatory statements dummy page
- Insert a location variable and enter it's embedded editor
- Insert an address variable inside the location variable
- Fill in the address variable
- Reload to check if all variables are correctly persisted


### Challenges/uncertainties
- On styling
   * When selecting an address variable inside a location variable and then leaving the location variable, the address variable still seems selected.
   * Unsure if it is clear enough to users which variable is selected (including the distinction between atomic and non-atomic variables)
- When a location variable only contains and address variable, the cursors don't work as expected. Including the firefox and chrome cursor plugins inside the embedded-editor gives weird results.



### Checks PR readiness
